mountpoint-s3/src/fuse.rs

```diff
@@ -92,28 +92,28 @@ where
         }
     }
 
-    #[instrument(level="warn", skip_all, fields(req=_req.unique(), ino=ino))]
+    #[instrument(level="warn", skip_all, fields(req=_req.unique(), ino=ino, name=field::Empty))]
```

Comment:
Why `name` and not `key`? Is there any case where we do not have the full key?

Reply:
Because, in the issue it was mentioned to add filename. So, I thought of adding filename wherever it is possible.

Reply:
Yes, you are right that this should be from the filesystem point of view. So ideally we could have the "full" path (from the mount point), but okay to stick to just the name for consistency and even more for brevity (the span is getting a bit busy).